### PR TITLE
fix all broken links in README

### PR DESCRIPTION
**Pull request by samlaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ The Registry Coordinator is a contract that is deployed by each AVS. It handles
 
 The registries are contracts that keep track of the attributes of individual operators. For example, we have initially built the
 
-- [StakeRegistry](./StakeRegistry.md) which keeps track of the stakes of different operators for different quorums at different times
-- [BLSPubkeyRegistry](./BLSPubkeyRegistry.md) which keeps track of the aggregate public key of different quorums at different times. Note that the AVS contracts use [BLS aggregate signatures](#bls-signature-checker) due to their favorable scalability to large operator sets.
-- [IndexRegistry](./IndexRegistry.md) which keeps track of an ordered list of the operators in each quorum at different times. (note that this behavior is likely only needed for EigenDA)
+- [StakeRegistry](./docs/middleware/StakeRegistry.md) which keeps track of the stakes of different operators for different quorums at different times
+- [BLSPubkeyRegistry](./docs/middleware/BLSPubkeyRegistry.md) which keeps track of the aggregate public key of different quorums at different times. Note that the AVS contracts use [BLS aggregate signatures](#bls-signature-checker) due to their favorable scalability to large operator sets.
```
Comment (ChaoticWalrus) [inline README.md:46]: these paths are pretty crappy. wouldn't it make sense to move all these docs into the main `/docs/` folder and delete the `/docs/middleware/` folder entirely?

Comment (stevennevins) [reply README.md]: agree here, moved them

Comment (ChaoticWalrus) [reply README.md]: ty ty

**Review by ChaoticWalrus — APPROVED** [review]
LGTM now. Thanks!

**Pull request merged**